### DISCUSSION TITLE
能不能增加微信h5支付啊, 手机端不好扫码啊

### DISCUSSION THREAD

**Discussion by qiudong19901008** [Baiyuetribe/kamiFaka#95] [新功能提交]
这发卡系统很不错, 就是手机支付时比较麻烦, 希望增加微信h5支付

**Comment by Baiyuetribe** [comment]
无人配合开发H5支付。条件要求：已备案域名+已开通微信H5

**Reply by qiudong19901008** [reply]
我可以配合.
我也会一点python, 但不知道怎么搞开源项目, 你可以告诉我怎么做

**Reply by Baiyuetribe** [reply]
很简单，不用管前端UI，你只需使用python代码复现一个最小的H5支付DEMO就行

**Comment by qiudong19901008** [comment]
我试试看

 

发件人: 佰阅 ***@***.***> 
发送时间: 2021年7月12日 19:38
收件人: Baiyuetribe/kamiFaka ***@***.***>
抄送: severe ***@***.***>; Author ***@***.***>
主题: Re: [Baiyuetribe/kamiFaka] 能不能增加微信h5支付啊, 手机端不好扫码啊 (#95)

 

很简单，不用管前端UI，你只需使用python代码复现一个最小的H5支付DEMO就行

**Comment by qiudong19901008** [comment]
写好了, 怎么提交给你, 地址是https://github.com/qiudong19901008/wxH5PayDemo
